FirebaseRemoteConfig/Sources/RCNConfigDBManager.h

```diff
@@ -78,7 +78,9 @@ typedef void (^RCNDBLoadCompletion)(BOOL success,
 /// Load Personalization from table.
 /// @param handler    The callback when reading from DB is complete.
 - (void)loadPersonalizationWithCompletionHandler:(RCNDBLoadCompletion)handler;
-
+/// Load Rollout metadata from table
+/// @param handler   The callback when reading from DB is complete.
+- (void)loadRolloutWithCompletionHandler:(RCNDBCompletion)handler;
```

Comment:
Same here `loadRolloutMetadataWithCompletionHandler` to make it more clear this isn't just one rollout

Reply:
Fixed.